uk.co.compucorp.civicrm.hrleaveandabsences/scss/components/outside-namespace/_leave-calendar.scss

```diff
@@ -20,12 +21,22 @@ $chr-leave-calendar-tooltip-item-min-width: 154px;
   }
 }
 
-.chr_leave-calendar__day-tooltip_item {
+.chr_leave-calendar__day-tooltip__item {
   cursor: pointer;
+  font-size: $chr-leave-calendar-tooltip-font-size;
   min-width: $chr-leave-calendar-tooltip-item-min-width;
   padding: $crm-gap-medium;
 
   &:hover {
     background-color: $gray-light;
   }
 }
+
+.chr_leave-calendar__day-tooltip__item_absence-type {
+  font-size: $font-size-base;
+}
+
+.chr_leave-calendar__day-tooltip__item_request-status {
+  color: $crm-gray-matte;
+  margin-bottom: #{$font-size-base - $chr-leave-calendar-tooltip-font-size};
```

Comment:
Not sure about this calculation using font size, what is it?

Reply:
I mentioned in in the PR description. This is to keep the gaps between the Status consistent. Otherwise due to `font-size` change the `line-height` changes as well, which makes upper gap be bigger than lower gap. This margin compensates this lack of gap. 

Note, we should not amend line-height in case Status appears on 2 lines.